modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -38,13 +38,17 @@ compute: <1> <2>
 - architecture: ppc64le
   hyperthreading: Enabled <3>
   name: worker
-  platform: {}
+  platform:
+    powervs:
+      smtLevel: 4 <4>
   replicas: 3
 controlPlane: <1> <2>
   architecture: ppc64le
   hyperthreading: Enabled <3>
   name: master
-  platform: {}
+  platform:
+    powervs:
+      smtLevel: 4 <4>
```

Comment:
why this value is set to 4 here? not sure what's the basic convention in the OCP, is it default value or something else?

Reply:
Updated to the default value 8.
cc: @mkumatag @juliemathew